app/src/components/v-checkbox.vue

```diff
@@ -238,5 +270,9 @@ function toggleInput(): void {
 		display: contents;
 		font-size: 1rem;
 	}
+
+	.append {
+		margin-left: 8px;
```

Comment:
Lets use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) whenever we can